rules/cloud/aws/aws_enum_logging.yml

```diff
@@ -0,0 +1,32 @@
+title: Potential Backup Enumeration on An AWS Instance
+id: 76255e09-755e-4675-8b6b-dbce9842cd2a
+status: experimental
+description: Detects potential enumeration activity targeting an AWS instance backups
+references:
+    - https://unit42.paloaltonetworks.com/compromised-cloud-compute-credentials/
+author: Janantha Marasinghe
+date: 2022/12/13
+tags:
+    - attack.discovery
+    - attack.t1580
+logsource:
+    product: aws
+    service: cloudtrail
+detection:
+    selection:
+        eventSource: 'ec2.amazonaws.com'
+        eventName:
+            - 'GetPasswordData'
+            - 'GetEbsEncryptionByDefault'
+            - 'GetEbsDefaultKmsKeyId'
+            - 'GetBucketReplication'
+            - 'DescribeVolumes'
+            - 'DescribeVolumesModifications'
+            - 'DescribeSnapshotAttribute'
+            - 'DescribeSnapshotTierStatus'
+            - 'DescribeImages'
+    timeframe: 10m
```

Comment:
timeframe is realy needed ?

Reply:
It's needed because the act of one event happening isn't suspicious on its own. For the rule to work it needs a correlation of events.

This rule will be rewritten to support the new PySIGMA correlation. For now, it's to use the old `timeframe` 

This applies to the all the rules that use timeframe here and in the repo